WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/RemoteOrderSynchronizer.swift

```diff
@@ -27,6 +27,8 @@ final class RemoteOrderSynchronizer: OrderSynchronizer {
 
     var setProduct = PassthroughSubject<OrderSyncProductInput, Never>()
 
+    var setProducts = PassthroughSubject<[OrderSyncProductInput], Never>()
```

Comment:
Same here about reusing `setProducts` and removing `setProduct`

Reply:
Same as https://github.com/woocommerce/woocommerce-ios/pull/9173#discussion_r1139845073 , simplifying can be addressed on https://github.com/woocommerce/woocommerce-ios/issues/9195